### PR TITLE
Bump max pin on `wrapt`

### PR DESCRIPTION
**Pull request by dhirschfeld**
## Steps

- [ ] For new features or bug fixes, add a ChangeLog entry describing what your PR does.
- [ ] Write a good description on what the PR does.

## Description

This PR bumps the max pin on `wrapt` now that version 1.14 has been released:
* https://github.com/GrahamDumpleton/wrapt/releases/tag/1.14.0
* https://wrapt.readthedocs.io/en/latest/changes.html#version-1-14-0

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

### CODE REVIEW

**Comment by agronholm** [comment]
Is there any reason for the version pin when other dependencies are mostly unpinned?

**Comment by dhirschfeld** [comment]
Happy to remove the max pin if the maintainers prefer.

Also, feel free to just fix this and merge - I've ticked the allow edits by maintainers box!

File: ChangeLog [hunk 1]
```diff
@@ -37,6 +37,8 @@ What's New in astroid 2.10.1?
 =============================
 Release date: TBA
 
+* Add support for wrapt 1.14
```
Comment (DanielNoord) [inline ChangeLog:40]: Could you put this in the `2.11` changelog? We're skipping `2.10.1` 😉

File: setup.cfg [hunk 2]
```diff
@@ -37,7 +37,7 @@ project_urls =
 packages = find:
 install_requires =
     lazy_object_proxy>=1.4.0
-    wrapt>=1.11,<1.14
+    wrapt>=1.11,<1.15
```
Comment (Pierre-Sassoulas) [inline setup.cfg:40]:
```suggestion
    wrapt>=1.11,<2
```
I think it's fairly safe to do.

**Review by Pierre-Sassoulas — APPROVED** [review]
Thank you for the fix ! And congratulation on becoming an astroid contributor :)

**Pull request merged**